### PR TITLE
bulk-action plugin updated

### PR DESCRIPTION
**Pull request by emreodabas**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/bulk-action.yaml is not a straightforward version bump: changing old version ("1.1.0-beta") with new version ("1.1.1") in the url (https://github.com/emreodabas/kubectl-plugins/archive/v1.1.0-beta.tar.gz) does not appear in the patch as https://github.com/emreodabas/kubectl-plugins/archive/v1.1.1.tar.gz

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/bulk-action.yaml is not a straightforward version bump: changing old version ("1.1.0-beta") with new version ("1.1.1") in the url (https://github.com/emreodabas/kubectl-plugins/archive/v1.1.0-beta.tar.gz) does not appear in the patch as https://github.com/emreodabas/kubectl-plugins/archive/v1.1.1.tar.gz

File: plugins/bulk-action.yaml [hunk 1]
```diff
@@ -21,11 +21,14 @@ spec:
   shortDescription: Do bulk actions on Kubernetes resources.
   caveats: |
     Usage:
-      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout) <command parameters>
+      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout|scale) <command parameters>
+
+    Help:
+      kubectl bulk-action help
```
Comment (corneliusweig) [inline plugins/bulk-action.yaml:27]: Can you move this to the `description` tag? `caveats` should be reserved for dependencies.

File: plugins/bulk-action.yaml [hunk 2]
```diff
@@ -21,11 +21,14 @@ spec:
...
     Usage:
-      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout) <command parameters>
+      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout|scale) <command parameters>
+
+    Help:
+      kubectl bulk-action help        
 
     This plugin needs the following programs:
     * grep
     * sed
```
Comment (corneliusweig) [inline plugins/bulk-action.yaml:31]: I think it's safe to assume that both programs are available on both supported platforms. So feel free to remove this from your `caveats` section.

A valid caveat would be, if you require the GNU version of these programs.

File: plugins/bulk-action.yaml [hunk 3]
```diff
@@ -21,11 +21,14 @@ spec:
...
   description: |
-    This plugin allows you to do bulk actions on Kubernetes resources.
+    This plugin allows you to do bulk actions on Kubernetes resources. Use carefully, its your own risk.
```
Comment (corneliusweig) [inline plugins/bulk-action.yaml:33]:
```suggestion
    This plugin allows you to do bulk actions on Kubernetes resources.
```
Isn't that obvious? Instead, you could say that your plugin will never ask for confirmations.

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/bulk-action.yaml is not a straightforward version bump: changing old version ("1.1.0-beta") with new version ("1.1.1") in the url (https://github.com/emreodabas/kubectl-plugins/archive/v1.1.0-beta.tar.gz) does not appear in the patch as https://github.com/emreodabas/kubectl-plugins/archive/v1.1.1.tar.gz

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/bulk-action.yaml is not a straightforward version bump: changing old version ("1.1.0-beta") with new version ("1.1.1") in the url (https://github.com/emreodabas/kubectl-plugins/archive/v1.1.0-beta.tar.gz) does not appear in the patch as https://github.com/emreodabas/kubectl-plugins/archive/v1.1.1.tar.gz

File: plugins/bulk-action.yaml (hunk 2 above)
Comment (emreodabas) [reply plugins/bulk-action.yaml]: It seems meaningful, I removed it.

**Pull request merged**